includes/definitions/discovery/uhp.yaml

```diff
@@ -0,0 +1,718 @@
+mib: UHP-MIB
+modules:
+  os:
+    hardware: ENTITY-MIB::entPhysicalDescr.1 
+    version: swVersion
+    serial: serialNumber   
```

Comment:
According to test data these don't work, or are null, maybe worth investigating a bit

Reply:
Good catch, thank you. Fixed this.

Reply:
Good @loopodoopo 
Now, the build fails because the test data is still broken compared to the code you fixed: 
You can check here: 
https://github.com/librenms/librenms/runs/8244891935?check_suite_focus=true

So just update the test data, and you'll be good to go.